src/main/java/org/gridsuite/study/server/dto/sensianalysis/nonevacuatedenergy/NonEvacuatedEnergyStagesSelection.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.dto.sensianalysis.nonevacuatedenergy;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Schema(description = "Sensitivity analysis non evacuated energy stages selection")
+public class NonEvacuatedEnergyStagesSelection {
+    String name;
+
```

Comment:
missing @Builder.Default ? There is a warning at the compilation

Reply:
Fixed